docs/plugin_development.rst

```diff
@@ -214,96 +266,122 @@ A minimal planning to organize ideas.
-#. In the main window of the recently created repository, click on the green "Code" button an copy the clone link.
-#. In your local command line run ``$ git clone [repo-link]``. You must have git installed and configured on your computer. If you don't have experience using git we recommend The Carpentries workshop `Version control with git`_.
+#. Click the :guilabel:`Create repository` button.
 
-A detailed description of `repository creation`_ could be found in the official Github documentation, and a `hello world`_ tutorial with basic git operations from the Github interface.
+#. In the main window of the newly created repository, click on the green :guilabel:`Code` button and copy the clone link.
 
-.. _Version control with git: https://swcarpentry.github.io/git-novice/
-.. _repository creation: https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories
-.. _hello world: https://docs.github.com/en/get-started/start-your-journey/hello-world
+#. On your local command line, run ``git clone <repo-link>``.
+   You must have Git installed and configured on your computer.
+   If you have no experience using Git, we recommend The Carpentries workshop `Version Control with Git`_.
 
+You can find a detailed description of `creating repositories`_ in the official GitHub documentation, and a `Hello World tutorial`_ with basic Git operations from the GitHub interface.
 
+.. _Version Control with Git: https://swcarpentry.github.io/git-novice/
+.. _creating repositories: https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories
+.. _Hello World tutorial: https://docs.github.com/en/get-started/start-your-journey/hello-world
 
-=================
-Let's get started
-=================
 
-We already have a git repository and a virtual environment where Spyder 5 is installed.
 
-Let's activate our environment and go into the local folder of our repository.
+.. _tutorial-plugin-structure:
+
+=============================
+Defining the plugin structure
+=============================
+
+We already have :ref:`a Git repository <tutorial-repo>` and :ref:`a virtual environment <tutorial-environment>` where Spyder 5 is installed.
+Let's activate our environment and go into the local folder of our repository:
 
 .. code-block:: shell
 
    mamba activate spyder-dev
    cd /path/to/your/repository
 
-Then we need to use ``cookiecutter`` to create the initial structure of our plugin. `cookiecutter`_ is a tool made in Python specifically designed to create project templates.
-We have developed one of these templates to generate the basic structure of a plugin, it can be found at: https://github.com/spyder-ide/spyder5-plugin-cookiecutter
+.. _tutorial-cookiecutter:
+
+Then we need to use `Cookiecutter`_ to create the :ref:`initial structure <tutorial-structure-details>` of our plugin.
+Cookiecutter is a tool made in Python specifically designed to create project templates.
```

Comment:
```suggestion
Cookiecutter is a tool implemented in Python specifically designed to create project skeletons.
```

Clarify some language—in this case, users aren't using it to create the original template but rather to produce the skeleton _from_ a template.